### PR TITLE
Update pwd stub file

### PR DESCRIPTION
**Pull request by LeMeteore**
Add the following properties to struct_passwd:
* n_fields
* n_sequence_fields
* n_unnamed_fields

To fix the following errors:
* pwd.struct_passwd.n_fields is not present in stub
* pwd.struct_passwd.n_sequence_fields is not present in stub
* pwd.struct_passwd.n_unnamed_fields is not present in stub

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
Could you reformat with Black?

**Comment by LeMeteore** [comment]
Yes I could reformat with Black --Nsukami_

**Comment by LeMeteore** [comment]
Is the Pillow error: 
```
stubtest failed for Pillow
error: PIL.Image.Image.__array__ is inconsistent, stub does not have argument "dtype"
```

somehow related to #5731 ?

**Comment by Akuli** [comment]
PIL error is unrelated to this PR. See #5746

File: stdlib/pwd.pyi [hunk 1]
```diff
@@ -8,6 +8,12 @@ class struct_passwd(Tuple[str, str, int, int, str, str, str]):
     pw_gecos: str
     pw_dir: str
     pw_shell: str
+    @property
+    def n_fields(self) -> int: ...
+    @property
+    def n_sequence_fields(self) -> int: ...
+    @property
+    def n_unnamed_fields(self) -> int: ...
```
Comment (srittau) [inline stdlib/pwd.pyi:16]: There are actually class properties:

```python
srittau@blitz:~$ python3
Python 3.7.3 (default, Jan 22 2021, 20:04:44) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pwd
>>> pwd.struct_passwd.n_fields
7
>>> 
```

**Review by srittau — CHANGES_REQUESTED** [review]

File: stdlib/pwd.pyi (hunk 1 above)
Comment (Akuli) [reply stdlib/pwd.pyi:16]: Maybe:
```suggestion
    n_fields: ClassVar[int]
    n_sequence_fields: ClassVar[int]
    n_unnamed_fields: ClassVar[int]
```

You will need to add `from typing import ClassVar`.

**Review by srittau — APPROVED** [review]

**Comment by Akuli** [comment]
You need to run `isort pwd.pyi`.

**Comment by LeMeteore** [comment]
Thanks a lot for your time, advices & suggestions :laughing:

**Pull request merged**